### PR TITLE
feat(docs): Add Chinese translations to docs

### PR DESCRIPTION
**Pull request by zsnmwy**

### CODE REVIEW

File: docs/tutorials/dynamic-scheduler-plugin.zh.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## 设计细节
+
+### 架构 
+![](./../images/dynamic-scheduler-plugin.png)
+
+
+如上图，动态调度器依赖于`Prometheus`和`Node-exporter`收集和汇总指标数据，它由两个组件组成：
+
+!!! note "Note"
```
[the remark this answers is not in the document]
Comment (zsnmwy) [reply docs/tutorials/dynamic-scheduler-plugin.zh.md:19]: What's the problem?  Syntax?

If you write markdown like:

```markdown
!!! note "Note"
    `Node-annotator` 目前是 `Crane-scheduler-controller`的一个模块.
```

The content will be render like:

image

Detail https://gocrane-crane-preview-pr-328.surge.sh/zh/tutorials/dynamic-scheduler-plugin/#_3

[the remark this answers is not in the document]
Comment (autumn0207) [reply docs/tutorials/dynamic-scheduler-plugin.zh.md:19]: ok, ignore it...I just see the render result in github markdown

File: docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md [hunk 2]
```diff
@@ -196,6 +196,248 @@ status:
...
+
+#### Horizontal scaling process
+
+cron 驱动和扩展过程有六个步骤：
+
+1. `EffectiveHPAController` 创建 `HorizontalPodAutoscaler`，它被注入到`spec`中的`external cron metrics`中。
+2. `HPAController` 从 `KubeApiServer` 读取 `external cron metrics`
+3. `KubeApiServer` 将请求转发给 `MetricAdapter` 和 `MetricServer`
+4. `MetricAdapter` 找到目标 hpa 的 `cron scaler`，并检测 `cron scaler` 是否处于活动状态。这意味着当前时间介于 cron 开始和结束计划时间之间。它将返回`TargetReplicas`中定义的`CronSpec`。
+5. `HPAController` 计算所有 metrics 结果，并通过选择最大的一个为目标副本数。并由此提出一个新的`scale replicas`。
```
Comment (zsnmwy) [inline docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md:342]:
```suggestion
5. `HPAController` 计算所有 metrics 结果，并通过选择最大的一个为目标副本数。并由此创建一个新的`scale replicas`。
```

File: docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md [hunk 3]
```diff
@@ -196,6 +196,248 @@ status:
...
 ```
 
+### Cron-based autoscaling
+EffectiveHorizontalPodAutoscaler 支持基于 cron 的自动缩放。
+
+除了基于监控指标，有时节假日和工作日的工作负载流量存在差异，简单的预测算法可能效果不佳。然后可以通过设置周末 cron 来支持更大数量的副本来弥补预测的不足。
+
+对于一些非 web 流量的应用，比如一些应用不需要在周末使用，可以把工作负载的副本数减少到 1，也可以配置 cron 来降低你的服务成本。
+
+以下是 EHPA 规范中的 cron 主要字段：
```
Comment (zsnmwy) [inline docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md:206]:
```suggestion
以下是 `EHPA Spec` 中的 cron 主要字段：
```

File: docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md [hunk 4]
```diff
@@ -196,6 +196,248 @@ status:
...
+
+- `CronSpec`：可以设置多个 cron 自动伸缩配置，cron cycle 可以设置循环的开始时间和结束时间，并且工作负载的副本数可以在时间范围内持续保证为设定的目标值。
```
Comment (zsnmwy) [inline docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md:208]:
```suggestion
- `CronSpec`：可以设置多个 cron 自动伸缩配置，cron cycle 可以设置循环的开始时间和结束时间，并且工作负载的副本数可以在时间范围内持续保持为设定的目标值。
```

File: docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md [hunk 5]
```diff
@@ -196,6 +196,248 @@ status:
...
+2. cron 的语义和行为不是很匹配，使用时甚至很难理解，很容易误导用户，导致自动伸缩失败。
+
+下图显示了当前 EHPA cron 自动伸缩实现与其他 cron 能力的对比。
+
+![crane-keda-ali-compare-cron.png](../images/crane-keda-ali-compare-cron.png)
+
+
+针对以上问题，EHPA 实现的 cron autoscaling 是在与 HPA 兼容的基础上设计的，cron 作为 HPA 的一个指标，与其他指标一起作用于工作负载。
+
+另外，cron 的设置也很简单。单独配置 cron 时，不在活动时间范围内时，不会执行工作负载的默认缩放。
```
Comment (zsnmwy) [inline docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md:227]:
```suggestion
另外，cron 的设置也很简单。单独配置 cron 时，不在活动时间范围内时，不会对工作负载执行缩放。
```

File: docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md [hunk 6]
```diff
@@ -196,6 +196,248 @@ status:
...
+  maxReplicas: 100       # MaxReplicas : autoscaler 缩放的最大副本数 
+  scaleStrategy: Auto   # ScaleStrategy : 缩放工作负载时候，所采用的策略。可选值为 "Auto" "Manual"
+  # 最好将Cron Scheduling设置为一个完整的时间周期，例如： 一天，一周
+  # 下面是一天的Cron Scheduling
+  #(targetReplicas)
+  #80                  --------     ---------        ----------
+  #                    |       |    |        |       |         |
+  #10       ------------       -----         --------          ----------
+  #(time)   0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23
+  #本地时区(timezone: "Local")意味着您使用Craned运行的服务器（或者可能是容器）的时区。例如，当Craned 是以UTC时区开始，那么它就是UTC。如果一开始是Asia/Shanghai，那么它就是Asia/Shanghai。
```
Comment (zsnmwy) [inline docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md:253]:
```suggestion
  #本地时区(timezone: "Local")意味着您使用运行Craned所在的服务器（或者可能是容器）的时区。例如，当Craned 是以UTC时区开始，那么它就是UTC。如果一开始是Asia/Shanghai，那么它就是Asia/Shanghai。
```

File: docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md [hunk 7]
```diff
@@ -196,6 +196,248 @@ status:
...
+4. `MetricAdapter` 找到目标 hpa 的 `cron scaler`，并检测 `cron scaler` 是否处于活动状态。这意味着当前时间介于 cron 开始和结束计划时间之间。它将返回`TargetReplicas`中定义的`CronSpec`。
+5. `HPAController` 计算所有 metrics 结果，并通过选择最大的一个为目标副本数。并由此提出一个新的`scale replicas`。
+6. `HPAController` 使用 `Scale Api` 缩放目标
+
+
+使用 EHPA 时，用户可以只配置 cron metric，让 EHPA 用作 cron hpa。
+
+一个 EHPA 的多个 crons 将转换为一个`external metrics`。
+
+HPA 将获取`external metrics`并在协调时计算目标副本。HPA 将选择最大的副本数来扩展来自多个指标的工作负载。
```
Comment (zsnmwy) [inline docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md:350]:
```suggestion
HPA 将获取`external metrics`并在协调时计算目标副本。当存在多个指标的工作负载时，HPA 将选择最大的副本数来扩展。
```

File: docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md [hunk 8]
```diff
@@ -196,6 +196,248 @@ status:
...
+kind: EffectiveHorizontalPodAutoscaler
+metadata:
+  name: php-apache-multi-dimensions
+spec:
+  # ScaleTargetRef 关联到需扩缩容的工作负载
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: php-apache
+  minReplicas: 1        # MinReplicas : 缩放的最少副本数
```
Comment (zsnmwy) [inline docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.zh.md:412]:
```suggestion
  minReplicas: 1        # MinReplicas : 缩放的最小副本数
```

**Comment by zsnmwy** [comment]
PTAL

Some Markdown features work only on the web. Like:  mermaid.

@autumn0207 @kitianFresh @qmhu

**Pull request merged**